postgres-connector/resources/test/docker/vanilla-schema-image/create_database.sql

```diff
@@ -1,1284 +0,0 @@
---
--- PostgreSQL database dump
---
```

Comment:
ditto on the license

Reply:
thats a tool to get your schema out of the database , I had to put it into the database in the first place